astropy/table/tests/test_operations.py

```diff
@@ -1460,9 +1460,9 @@ def test_mixin_functionality(self, mixin_cols):
         else:
             with pytest.raises(NotImplementedError) as err:
                 table.vstack([t, t])
-            assert "vstack unavailable for mixin column type(s): {}".format(
-                cls_name
-            ) in str(err.value)
+            assert f"vstack unavailable for mixin column type(s): {cls_name}" in str(
+                err.value
+            )
```

Comment:
If we are updating this code we should do it properly using `pytest.raises(..., match=...)`.

Reply:
✅